### PR TITLE
Add stub pages for methodology

### PR DESCRIPTION
**Pull request by craig-statsig**

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://635ae2e09fd7350baece3747--cozy-fox-0defec.netlify.app

File: docs/stats-engine/methodologies/bonferroni-correction.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: Bonferroni Correction
+sidebar_label: Bonferroni Correction
+slug: /stats-engine/methodologies/bonferroni-correction
+---
+
+## Bonferroni Correction
+
+If you run a tests with α = 0.05, the probability of a false positive will be 5%. If you run more comparisons at the same significance level, the chance of at least one false positive goes up because each comparison is an additional opportunity for false positive.
+
+Bonferonni corrections are an optional feature on Statsig experiments that reduces the probability of Type I errors (false positives) by adjusting the significance level (α). The significance level is divided by the number of comparisons (equivalent to the number of test variants) in the experiment.
```
Comment (tim-statsig) [inline docs/stats-engine/methodologies/bonferroni-correction.md:11]:
```suggestion
Bonferroni corrections are an optional feature on Statsig experiments that reduces the probability of Type I errors (false positives) by adjusting the significance level (α). The significance level is divided by the number of comparisons (equivalent to the number of test variants) in the experiment.
```

File: docs/stats-engine/methodologies/cuped.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: CUPED
+sidebar_label: CUPED
+slug: /stats-engine/methodologies/cuped
+---
+
+## Delta Method
```
Comment (tim-statsig) [inline docs/stats-engine/methodologies/cuped.md:7]:
```suggestion
## CUPED
```

File: docs/stats-engine/methodologies/cuped.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+CUPED is a technique Statig uses in which we use information about an experiment's users from before an experiment started to reduce the variance in their experiment metrics. This also can help to debias experiments which have meaningful pre-exposure bias (e.g. the groups were randomly different before any treatment was applied).
```
Comment (tim-statsig) [inline docs/stats-engine/methodologies/cuped.md:9]:
```suggestion
CUPED (short for Controlled-experiment Using Pre-Existing Data) is a technique Statig uses in which we use information about an experiment's users from before an experiment started to reduce the variance in their experiment metrics. This also can help to debias experiments which have meaningful pre-exposure bias (e.g. the groups were randomly different before any treatment was applied).
```

File: docs/stats-engine/methodologies/delta-method.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+title: Delta Method
+sidebar_label: Delta Method
+slug: /stats-engine/methodologies/delta-method
+---
+
+## Delta Method
+
+Statsig uses the delta method when calculating the variance for variables that have a numerator and denominator.
+
+The variance of ratio and mean metrics depends upon the numerator and denominator variables, which are typically correlated. For example, consider a _clicks per session_ metric. The number of clicks and the number of sessions are two sets of obervations coming from the same group of users, so they are not independent from each other. To properly account for this correlation, the variance of the mean of a ratio metric _R_ is obtained using the delta method:
```
Comment (tim-statsig) [inline docs/stats-engine/methodologies/delta-method.md:11]:
```suggestion
The variance of ratio and mean metrics depends upon the numerator and denominator variables, which are typically correlated. For example, consider a _clicks per session_ metric. The number of clicks and the number of sessions are two sets of observations coming from the same group of users, so they are not independent from each other. To properly account for this correlation, the variance of the mean of a ratio metric _R_ is obtained using the delta method:
```

File: docs/stats-engine/methodologies/srm-checks.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+---
+title: SRM Checks
+sidebar_label: SRM Checks
+slug: /stats-engine/methodologies/srm-checks
+---
+
+## Sample Ratio Mismatch
```
Comment (tim-statsig) [inline docs/stats-engine/methodologies/srm-checks.md:7]:
```suggestion
## SRM - Sample Ratio Mismatch
```

File: docs/stats-engine/methodologies/srm-checks.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+Sample ratio mismatch is when the observed allocation of users between test groups differs from the expected allocation or "split" of the test.
```
Comment (tim-statsig) [inline docs/stats-engine/methodologies/srm-checks.md:9]:
```suggestion
Sample ratio mismatch (SRM for short) is when the observed allocation of users between test groups differs from the expected allocation or "split" of the test.
```

**Pull request merged**